commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/CaseUtils.java

```diff
@@ -0,0 +1,252 @@
+package org.commcare.commcaresupportlibrary;
```

Comment:
I'm presuming these were copied over ~identically from the support library and don't need new review, is that assumption correct?

Reply:
yes. It's an exact copy so no need to review.

Reply:
well one [minor change](https://github.com/dimagi/commcare-android/pull/2407/commits/b57827b47f104aa5e2db3bb5c5cadde36d0cad04) that I had to do is target the same android versions as `commcare-android` in order to build successfully and avoid some resource conflicts.